### PR TITLE
Enable prepull for k8s 1.11+ containerd test.

### PR DESCRIPTION
**Pull request by Random-Liu**
https://github.com/kubernetes/kubernetes/issues/55141 is fixed. We should be able to enable prepull now.
/cc @yujuhong 

Signed-off-by: Lantao Liu <lantaol@google.com>

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -92,6 +92,7 @@
       "--check-leaked-resources",
```
Comment (BenTheElder) [inline jobs/config.json]: we should move all of this to another owned directory with you and yuju as OWNERS :-)

**Comment by krzyzacy** [comment]
@Random-Liu ^^ do you want to do in this PR, or as a follow up?

/lgtm
/hold

**Comment by Random-Liu** [comment]
@krzyzacy I'll do it in another PR. Will send out today.

**Comment by krzyzacy** [comment]
ack, feel free to /hold cancel this one

**Pull request merged**